server/service/src/sync/test/test_data/master_list.rs

```diff
@@ -65,6 +105,37 @@ pub(crate) fn test_pull_upsert_records() -> Vec<TestSyncPullRecord> {
                 description: "note 2".to_owned(),
             }),
         ),
+        TestSyncPullRecord::new_pull_upserts(
+            LegacyTableName::LIST_MASTER,
+            MASTER_LIST_3,
+            vec![
+                PullUpsertRecord::MasterList(MasterListRow {
+                    id: MASTER_LIST_3.0.to_owned(),
+                    name: "program_test".to_owned(),
+                    code: "".to_owned(),
+                    description: "note 3".to_owned(),
+                }),
+                PullUpsertRecord::Program(ProgramRow {
+                    id: MASTER_LIST_3.0.to_owned(),
+                    name: "program_test".to_owned(),
+                }),
+                PullUpsertRecord::ProgramRequisitionSettings(ProgramRequisitionSettingsRow {
+                    id: "program_test".to_owned() + &mock_name_tag_1().id,
```

Comment:
```suggestion
                    id: MASTER_LIST_3.0.to_owned() + &mock_name_tag_1().id,
```
More clearly communicates intent to me...